src/huggingface_hub/utils/_git_credential.py

```diff
@@ -37,9 +37,15 @@ def list_credential_helpers(folder: Optional[str] = None) -> List[str]:
         # NOTE: If user has set an helper for a custom URL, it will not we caught here.
         #       Example: `credential.https://huggingface.co.helper=store`
         #       See: https://github.com/huggingface/huggingface_hub/pull/1138#discussion_r1013324508
+        for line in output.split("\n"):
+            if "credential.helper" in line:
+                if line.split("=")[-1]:
+                    value = line.split("=")[-1].split()[0]
+                else:
+                    value = line.split("=")[-1].strip()
         return sorted(  # Sort for nice printing
             {  # Might have some duplicates
-                line.split("=")[-1].split()[0] for line in output.split("\n") if "credential.helper" in line
+                value
```

Comment:
We must return a set of all git credential helpers configured on the machine. So here we would need to return a value for each line with `"credential.helper="` in it.

Reply:
I see, thanks for pointing it out.